examples/chopsticks/README.md

```diff
@@ -4,7 +4,7 @@ This example demonstrates how to use [chopsticks](https://github.com/AcalaNetwor
 ## Setup local acala fork with chopsticks
 - run a local fork of Acala (use `npx` or `yarn dlx` if you prefer these over `bunx`)
 ```
-bunx @acala-network/chopsticks@latest -c https://raw.githubusercontent.com/AcalaNetwork/chopsticks/master/configs/acala.yml
+bunx @acala-network/chopsticks@latest -c acala
```

Comment:
does bunx works for you? I get `DriverPackageNotInstalledError: SQLite package has not been found installed.`

Reply:
upgrade to the latest version https://github.com/oven-sh/bun/issues/7593

Reply:
yup bun `1.0.17` works for me